src/plugins/intel_gpu/src/runtime/kernels_cache.hpp

```diff
@@ -22,9 +23,31 @@
 #include "ocl/ocl_engine.hpp"
 
 namespace cldnn {
+using cached_kernel_id_type = std::pair<size_t, std::string>;
```

Comment:
As a note: The only thing I have in mind as a concern is the potential hash collision of the key, 
because not only "entry_key" but also the hash from the binary. 
From the offline conversation with eddy, he can replace the key without binary hash.

Reply:
I updated not to use hash. Thank you.

Reply:
@e-ddykim , @yeonbok Updated commits are merged to this branch now. thanks.